conf/aggregations.go

```diff
@@ -85,6 +91,38 @@ func ReadAggregations(file string) (Aggregations, error) {
 			}
 		}
 
+		writeBufferStr := s.ValueOf("writeBuffer")
```

Comment:
>  so when the write buffer is instantiated for the rollups

what exactly do you mean with rollups? The AggMetric representing a rollup for another AggMetric? that one does not have the reorder window. the reordering only happens on the "main" AggMetric (not any instantiated by its Aggregators) which I think is correct.